src/app/issuer/components/issuer-staff/issuer-staff.component.html

```diff
@@ -182,5 +333,5 @@ <h2>{{ 'Issuer.addMember_failed' | translate }}!</h2>
 				</tr>
 			</tbody>
 		</table>
-	</div>
+	</div> -->
```

Comment:
If it is not necessary anymore, I'd say remove it instead of leaving it in as a comment.
we can always go back with git history :)